src/app/modules/training-program/components/programs/program-list.component.html

```diff
@@ -1,31 +1,12 @@
 <div class="container-fluid">
-  <div class="row">
-    <div class="col-12">
-      <div class="card">
+  <div class="row" *ngFor="let programDetail of programs">
```

Comment:
Use `program` instead of `programDetail`, it needs to be consistent given the ctx.